File: code-samples/messaging/global-business/cancel-message.cs
using System;
using com.Messente.Api.Api;
using com.Messente.Api.Client;

namespace Example
{
    public class CancelMessage
    {
        public static void Main()
        {
            Configuration conf = new Configuration
            {
                Username = "YOUR_MESSENTE_API_USERNAME",
                Password = "YOUR_MESSENTE_API_PASSWORD"
            };

            var apiInstance = new OmnimessageApi(conf);

            try
            {
                apiInstance.CancelScheduledMessage("<omnimessage_id>");
                Console.WriteLine("Scheduled omnimessage cancelled");
            }
            catch (Exception e)
            {
                Console.WriteLine("Exception when cancelling an omnimessage: " + e.Message);
            }
        }
    }
}

File: code-samples/messaging/global-business/cancel-message.py
from messente_api import ApiClient, Configuration, OmnimessageApi
from messente_api.rest import ApiException

configuration = Configuration()
configuration.username = "YOUR_MESSENTE_API_USERNAME"
configuration.password = "YOUR_MESSENTE_API_PASSWORD"

api_instance = OmnimessageApi(ApiClient(configuration))

try:
    api_instance.cancel_scheduled_message("<omnimessage_id>")
    print("Scheduled omnimessage cancelled")
except ApiException as exception:
    print("Exception when cancelling an omnimessage: %s\n" % exception)
